deployment/ConnectedVehicleARMTemplate.json

```diff
@@ -74,6 +74,28 @@
             "metadata": {
                 "description": "Claims provider uri which is used for platformAccount deployment."
             }
+        },
+        "logAnalyticsWorkspaceName": {
+            "type": "string",
+            "defaultValue": "[concat(uniqueString(resourceGroup().id), 'mcvplogs')]",
+            "metadata": {
+                "description": "The name of the log analytics workspace to configure diagnostic settings for MCVP."
+            }
+        },
+        "logAnalyticsWorkspaceSku": {
+            "type": "string",
+            "allowedValues": [
+                "pergb2018",
+                "Free",
+                "Standalone",
+                "PerNode",
+                "Standard",
+                "Premium"
+            ],
+            "defaultValue": "pergb2018",
+            "metadata": {
+                "description": "Pricing tier: PerGB2018 or legacy tiers (Free, Standalone, PerNode, Standard or Premium) which are not available to all customers."
```

Comment:
The description confused me. What is not available to the customers?

Reply:
Good point. This was copied over from the sample template. I will update it.

Reply:
Confirmed all in the list is available. 

https://docs.microsoft.com/en-us/azure/templates/microsoft.operationalinsights/2020-03-01-preview/workspaces?tabs=bicep#workspacesku